suites/pacific/rgw/sanity_rgw.yaml

```diff
@@ -0,0 +1,36 @@
+# RHCS 5.0 sanity test suite for RGW daemon.
+tests:
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: setup pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        registry: false
+        deployment: true
+      desc: RHCS cluster deployment using cephadm.
+      destroy-cluster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      config:
+        exec_shell:
+          - 'ceph -s'
+      desc: Verify the health of the storage cluster.
+      destroy-cluster: false
+      module: test_cephadm.py
+      name: verify cluster health
+
+  - test:
+      abort-on-fail: true
+      config:
+        branch: master
```

Comment:
do you want to use `master` or`pacific`. I think its best we use `pacific` branch here. cause `master` always points to latest release. Because of this we may have to update this value in the future, which we may forget :)

Reply:
Ideally, it should be `ceph-pacific`. At the time of pushing the PR, the branch was not created hence I left it as `master`. Now that there is a `ceph-pacific` branch, I'll modify it.